modules/social_features/social_group/social_group.module

```diff
@@ -1188,6 +1192,43 @@ function _social_group_cancel_join_leave_form(array $form, FormStateInterface $f
   }
 }
 
+/**
+ * Validation callback for group content create form.
+ *
+ * @param array $form
+ *   Group node form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   Form state interface.
+ */
+function _social_group_node_form_cross_posting_validate(array $form, FormStateInterface $form_state) {
+  if (empty($form_state->getValue('groups'))) {
+    return;
+  }
+  // Cross posting is disabled globally.
+  $group_settings = \Drupal::config('social_group.settings');
+  if (!$group_settings->get('cross_posting.status')) {
+    return;
+  }
+  // No group types disabled for cross-posting.
```

Comment:
@nechai It's better to rename name and logic of this variable from `disabled` to `enabled`